CHANGELOG.md

```diff
@@ -3,6 +3,39 @@
 Entries are listed in reverse chronological order per undeprecated
 major series.
 
+## 4.x series
+
+### 4.0.0
+
+#### Breaking changes
+
+* Update the MSRV from 1.41 to 1.56.1
+* Make `digest` an optional feature
+* Make `rand_core` an optional feature
+* Add target u32/u64 backend overrides
+* Update backend selection to be more automatic
+* Remove `std` feature flag
+* Remove `nightly` feature flag
+* Replace methods `Scalar::{zero, one}` with constants `Scalar::{ZERO, ONE}`
```

Comment:
`MINUS_ONE` is not in the public API of `Scalar`. I had to recompile docs to check that. Also docs are broken, so will fix.

Reply:
It's hard to tell sometimes, due to the existing codebase using `pub fn` for methods on private types. A nice cleanup would be to make those all `pub(crate) fn` so that visibility is more obvious locally.